src/pages/CreateEvent/EventForm.tsx

```diff
@@ -163,230 +232,373 @@ const EventForm: React.FC = () => {
-            <p className="error-text">{errors.announcementDate.message}</p>
-          )}
+          {errors.endDate && <p className="error">{errors.endDate.message}</p>}
+          {errors.endTime && <p className="error">{errors.endTime.message}</p>}
         </div>
 
         <div className="form-group">
-          <label>Timezone</label>
+          <label>Timezone<sup>*</sup></label>
           <select
             {...register("timezone", { required: "Timezone is required" })}
+            className={`form-control${errors.timezone ? " error" : ""}`}
           >
+            <option value="">Select a timezone</option>
             <option value="GMT-0700">PST (GMT-0700)</option>
+            <option value="GMT-0600">MST (GMT-0600)</option>
+            <option value="GMT-0600">CST (GMT-0600)</option>
             <option value="GMT-0500">EST (GMT-0500)</option>
           </select>
+          {errors.timezone && (
+            <p className="error">{errors.timezone.message}</p>
+          )}
         </div>
 
         <div className="form-group">
-          <label>Meeting Link *</label>
-          <input type="url" {...register("meetingLink")} />
+          <label>Meeting Link<sup>*</sup></label>
+          <input
+            type="url"
+            {...register("meetingLink", {
+              required: "Meeting Link is required",
+            })}
+            className={`form-control${errors.meetingLink ? " error" : ""}`}
+          />
+          {errors.meetingLink && (
+            <p className="error">{errors.meetingLink.message}</p>
+          )}
         </div>
 
         <div className="form-group">
-          <label>Participant Count *</label>
-          <input
-            type="number"
-            {...register("minParticipants", { valueAsNumber: true })}
-            placeholder="Min"
-          />{" "}
-          -
-          <input
-            type="number"
-            {...register("maxParticipants", { valueAsNumber: true })}
-            placeholder="Max"
-          />
+          <label>Participant Count<sup>*</sup></label>
+          <div className="flex flex-col">
+            <label>Min</label>
+            <input
+              type="number"
+              {...register("minParticipants", {
+                valueAsNumber: true,
+                required: "Minimum participant count is required",
+                min: {
+                  value: 1,
+                  message: "Minimum participants must be at least 1",
+                },
+              })}
+              placeholder="4"
+              className={`w-2/12 p-1 rounded text-base ${
+                errors.minParticipants ? "error" : ""
+              }`}
+            />
+            {errors.minParticipants && (
+              <p className="error">{errors.minParticipants.message}</p>
+            )}
+          </div>
+          <div className="flex flex-col">
+            <label>-</label>
+            <label>Max</label>
+            <input
```

Comment:
min number available should be at least the minParticipantCount + 1

Reply:
Done

Reply:
<img width="425" alt="image" src="https://github.com/user-attachments/assets/2b8b0b6d-9da1-4bce-b989-9ecd8c525752">


Weird issue happens 🤔

Reply:
Fixed, forgot a '=' sign...